### ISSUE TITLE
ML

### ISSUE THREAD

**Issue by bthek1** [bthek1/ML#1]
- [x] #2
- [x] #3
- [x] #4
- [x] #5
- [x] #6
- [x] #7
- [x] #8
- [x] #9
- [x] #10
- [x] #11
- [x] #12

**Comment by bthek1** [comment]
- [ ] Cross Entropy
- [ ] Back Propagation
- [ ] Convolution Neural Networks
- [ ] Ensemble Methods: Bagging and Boosting
- [ ] Non-linear Input Transformations and Kernels
- [ ] The Bayesian Approach and Gaussian Processes
- [ ] Generative Models and Learning from Unlabelled Data
- [ ] Support Vector Regression
- [ ] Random Forest
- [ ] Back Propagation and Gradient Descent

**Comment by bthek1** [comment]
- [] PCA
